osf/management/commands/start_sloan_phase.py

```diff
@@ -0,0 +1,79 @@
+"""
+Switches Sloan user tags, Switches and Flags into a state where the Sloan study can begin.
+"""
+import logging
+from osf.models import (
+    Tag,
+)
+
+from waffle.models import (
+    Flag,
+    Switch
+)
+from django.utils import timezone
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+logger = logging.getLogger(__name__)
+
+WAFFLE = 'waffle:sloan|{}'
+NO_WAFFLE = 'no_waffle:sloan|{}'
+FLAG_NAME = 'sloan_{}_display'
+SWITCH_NAME = 'sloan_{}_input'
+AVAILABLE_FLAGS = ['coi', 'data', 'prereg']
+
+
+def flip(flag):
+    try:
+        waffle_tag = Tag.all_tags.get(system=True, name=WAFFLE.format(flag))
+        waffle_tag.osfuser_set.clear()
+        print(f'Tag {WAFFLE.format(flag)} cleared')
+    except Tag.DoesNotExist:
+        print(f'Tag {WAFFLE.format(flag)} not found')
+        pass
+    try:
+        no_waffle_tag = Tag.all_tags.get(system=True, name=NO_WAFFLE.format(flag))
+        no_waffle_tag.osfuser_set.clear()
+        print(f'Tag {NO_WAFFLE.format(flag)} cleared')
+    except Tag.DoesNotExist:
+        print(f'Tag {NO_WAFFLE.format(flag)} not found')
+        pass
+    waffle_flag, created = Flag.objects.get_or_create(name=FLAG_NAME.format(flag), percent=50.0, everyone=None)
```

Comment:
This won't match a `get` unless all three fields match, which could lead to duplicate `Flag`'s. `Flag.objects.update_or_create(name=FLAG_NAME.format(flag), defaults={'percent': 50.0, 'everyone':None})` would obviate the next three lines

Likewise for `Switch` below.